src/nimblepkg/publish.nim

```diff
@@ -206,7 +206,7 @@ proc publish*(p: PackageInfo, o: Options) =
     if parsed.scheme == "":
       # Assuming that we got an ssh write/read URL.
       let sshUrl = parseUri("ssh://" & url)
-      url = "https://github.com/" & sshUrl.port & sshUrl.path
+      url = "https://" & sshUrl.hostname & "/" & sshUrl.port & sshUrl.path
```

Comment:
Nice catch.

I'm pretty sure this URL construction will still fail if a port is specified though. But i'll merge anyway.